apps/src/codebridge/FileTabs/FileTab.tsx

```diff
@@ -17,30 +20,31 @@ const FileTab = ({file}: FileTabProps) => {
   const activeFile = getActiveFileForProject(project);
   const {iconName, iconStyle, isBrand} = getFileIconNameAndStyle(file);
   const iconClassName = isBrand ? 'fa-brands' : undefined;
+  const isActive = file.active || file === activeFile;
+  const className = isActive
+    ? classNames(moduleStyles.fileTab, moduleStyles.active)
+    : moduleStyles.fileTab;
 
   return (
-    <div
-      className={`${moduleStyles.fileTab} ${
-        file.active || file === activeFile ? moduleStyles.active : ''
-      }`}
-      key={file.id}
-    >
+    <div className={className} key={file.id}>
       <span onClick={() => setActiveFile(file.id)}>
         <FontAwesomeV6Icon
           iconName={iconName}
           iconStyle={iconStyle}
           className={iconClassName}
         />
-        &nbsp;
-        {file.name}
+        <span>{file.name}</span>
```

Comment:
Are we meant to use the Typography DSCO element here? The figma mentions H6

Reply:
The figma does mention H6 but an H6 was way too big 🤷‍♀️ I think the figma assumes we can use the Tab component but we cannot because it's not closeable.

Reply:
ahh, ok then! Looks good :D